.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,6 +9,8 @@ about: Create a report to help reproduce and fix the issue
 If you discovered this issue from playing tgstation hosted servers:
 [Round ID]: # (It can be found in the Status panel or retrieved from https://sb.atlantaned.space/rounds ! The round id let's us look up valuable information and logs for the round the bug happened.)-->
 
+<!-- If you are reporting an issue found in another branch or codebase, you MUST link the branch or codebase repo in your issue report or it will be closed. For branches, If you have not pushed your code up, you must either repo it on master or push your code up before making an issue report. For other codebases, if you do not have a public code repo you will be refused help unless you can completely repo the issue on our code. -->
```

Comment:
```suggestion
<!-- If you are reporting an issue found in another branch or codebase, you MUST link the branch or codebase repo in your issue report or it will be closed. For branches, If you have not pushed your code up, you must either repo it on master or push your code up before making an issue report. For other codebases, if you do not have a public code repo you will be refused help unless you can completely reproduce the issue on our code. -->
```